extensions/HuggingFaceTransformers/python/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools", "wheel"]
 
 [project]
 name = "aiconfig_extension_hugging_face_transformers"
-version = "0.0.4"
+version = "0.0.5"
```

Comment:
open question but should we be including all modalities into one package/extension or have a seperate extension for each?

Reply:
I think for now let's just do all of them in one package since this is goaled for the Gradio component. We can discuss this in the future: https://github.com/lastmile-ai/aiconfig/issues/480